cabby/geo/sample_poi.py

```diff
@@ -56,8 +56,8 @@ def main(argv):
     FLAGS.region, FLAGS.min_s2_level, FLAGS.directory)
 
```

Comment:
Make sure to merged with master and update this to use the new region stuff that was committed a bit ago, e.g.:

```
map_region = map_structure.Map(
    regions.get_region(FLAGS.region), FLAGS.min_s2_level, FLAGS.directory)	 
```

Reply:
Already did :)